src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -22,6 +22,7 @@ namespace System.Security.Cryptography.Pkcs.EnvelopedCmsTests.Tests
     public static partial class DecryptTests
```

Comment:
That idea makes me nervous – I’d rather see the attribute directly on the method where I’m more likely to notice it and know it’s not gonna get run locally.